blockly-core/core/ui/block.js

```diff
@@ -254,14 +254,14 @@ Blockly.Block.isFreelyDragging = function() {
 
 /**
  * Wrapper function called when a mouseUp occurs during a drag operation.
- * @type {Array.<!Array>}
+ * @type {bindData}
```

Comment:
If bindData is a class or custom type (e.g. typedef), I think it should be called BindData to avoid confusion with variable names.
   
Where is BindData declared?